### DISCUSSION TITLE
Chrome Core Compatibility Issue: No H.264 Support for Online Video Playback

### DISCUSSION THREAD

**Discussion by yuaotianAI** [jcefmaven/jcefmaven#82] [Q&A]
Hi, may I ask if there is a Chrome core that supports H.264 encoding? I'm using the latest Maven version, and I found that it doesn't support H.264 when testing on [html5test.com](https://html5test.com/). This means it doesn't support playing online videos like MP4.

**Comment by FriwiDev** [comment]
Sadly there are no public chrome distributions available for download including these proprietary codecs, as distributing them makes you obliged to pay a fee. If you wish to enable them you need to compile an own version of chromium with the codecs enabled and pay the fee when distributing.

Am 14.09.2023 um 10:29 schrieb 徐长卿 ***@***.***>:

﻿

Hi, may I ask if there is a Chrome core that supports H.264 encoding? I'm using the latest Maven version, and I found that it doesn't support H.264 when testing on html5test.com. This means it doesn't support playing online videos like MP4.
